.github/workflows/build.yml

```diff
@@ -73,3 +74,11 @@ jobs:
       # Ensure we don't break scripts
       - name: Build utils
         run: npm run build-utils
+
+      # No need to run the analysis from all environments
+      - name: SonarCloud Scan
+        if: ${{ success() && contains(matrix.os.coverage, 'coverage') }}
+        uses: SonarSource/sonarcloud-github-action@master
```

Comment:
:warning: I would prefer we use a tag version of the action to avoid any unwanted side effects on new release of the action
This way of doing comes from the action documentation: https://github.com/SonarSource/sonarcloud-github-action

Reply:
Something like that ?
```suggestion
        uses: SonarSource/sonarcloud-github-action@1.4
```

Reply:
![image](https://user-images.githubusercontent.com/4921914/100359349-b41a9280-2ff7-11eb-952d-019e04ee2ccf.png)

Reply:
yes